pytorch_lightning/accelerators/accelerator.py

```diff
@@ -174,7 +175,9 @@ def batch_to_device(
         """
         model = self.lightning_module
 
-        if model is not None:
+        # TODO: Add support to allow batch transfer to device in Lightning for DP mode.
```

Comment:
Already have this one

https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pytorch_lightning/trainer/configuration_validator.py#L111